### DISCUSSION TITLE
Can I make a clickable Image in the visual editor?

### DISCUSSION THREAD

**Discussion by jenny787** [requarks/wiki#6501] [Help / Questions] [answered]
How can I have an image be clickable, which takes you to a url?  I cannot figure it out.

**Comment by P4mes4n** [comment]
You can do it like this:
`[![Picture with 16x16 Pixel](/assets/site/picture.png =16x)](/somePage)`

**Reply by NGPixel** [reply]
That's Markdown, the question was for the Visual Editor.

**Reply by P4mes4n** [reply]
Oh, I missed that. Sorry!

**Comment by NGPixel** [comment, marked as answer]
I don't think it's possible. It will be addressed in v3.
